src/codegen.cpp

```diff
@@ -105,6 +105,10 @@
 #if defined(_CPU_ARM_) || defined(_CPU_AARCH64_)
 #  include <llvm/IR/InlineAsm.h>
 #endif
+#if defined(USE_POLLY)
+#include <polly/RegisterPasses.h>
+#include <polly/ScopDetection.h>
```

Comment:
I'm now going to think "Scop" is a typo every time I see it. "Static control part," apparently? https://github.com/llvm-mirror/polly/blob/master/include/polly/ScopDetection.h